deployments/add-ons/prometheus/config-map.yaml

```diff
@@ -0,0 +1,172 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: prometheus-server-conf
+  labels:
+    name: prometheus-server-conf
+  namespace: monitoring
+data:
+  prometheus.rules: |-
+    groups:
+    - name: devopscube demo alert
+      rules:
+      - alert: High Pod Memory
+        expr: sum(container_memory_usage_bytes) > 1
+        for: 1m
+        labels:
+          severity: slack
```

Comment:
Do you have a reference to the different severity values?

Reply:
These are the default values in the prometheus community helm charts:
https://prometheus-community.github.io/helm-charts